workitem/workitem.go

```diff
@@ -18,6 +18,8 @@ type WorkItem struct {
 	Version int
 	// the field values
 	Fields Fields `sql:"type:jsonb"`
+	// the position of workitem
+	Executionorder float64
```

Comment:
@DhritiShikhar I think what @aslakknutsen meant is that you should use an uppercase `O`, right? So `Executionorder` becomes `ExecutionOrder`.